### DISCUSSION TITLE
Is a .net5 mvc app still supported by latest version (5.5.0)?

### DISCUSSION THREAD

**Discussion by elbulla** [akunzai/GSS.Authentication.CAS#656] [General]
I'm so far not able to get the client to work properly with my net5 mvc app. It uses a Startup.cs, so I followed the AspNetCoreSample sample setup for 5.5.0, but moved things as need to my Startup.cs ConfigureServices method. I see looking back at previous version that there used to be an MVC sample project back in 4.0. Do I need to use that version?

What I get so far is an infinite redirect loop to Account/Login when I startup the app:

http://localhost:63367/Account/Login?ReturnUrl=%2FAccount%2FLogin%3FReturnUrl%3D%252FAccount%252FLogin...

Any hints of other things I may need to adjust to work with net5 mvc?

Thank you!

**Comment by akunzai** [comment]
Yes, It still supported by latest version

Take look this [ASP.NET Core MVC Sample](https://github.com/akunzai/GSS.Authentication.CAS/blob/main/samples/AspNetCoreMvcSample).

Remember to add the `[AllowAnonymous]` attribute on [AccountController](https://github.com/akunzai/GSS.Authentication.CAS/blob/main/samples/AspNetCoreMvcSample/Controllers/AccountController.cs#L7).

**Comment by elbulla** [comment]
Thank you so much, had to make slight adjustments since this is also a Razor pages project, but got it to work!
